lib/bitstyles_phoenix/component/form.ex

```diff
@@ -547,10 +625,44 @@ defmodule BitstylesPhoenix.Component.Form do
     <%= PhxForm.label @form, @field, @label_opts do %>
       <%= render_slot(@inner_block) %>
       <%= @label %>
+      <.required_label {assigns_to_attributes(assigns)} />
     <% end %>
     <.ui_errors form={@form} field={@field} />
     """
   end
 
   defp default_label(field), do: PhxForm.humanize(field)
+
+  defp required_label(%{required: value} = assigns)
+       when not is_nil(value) do
```

Comment:
Just wondering: 
Would it also make sense to check that the value is not `false`?